thirdeye/thirdeye-frontend/app/styles/components/contribution-table.scss

```diff
@@ -0,0 +1,129 @@
+/* analysis table css*/
```

Comment:
copied from old UI. Mostly following BEM syntax. 
Will need to clean up after the launch of this MVP